MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -1261,6 +1300,163 @@ void EnggDiffractionViewQtGUI::setListWidgetBank(int idx) {
   selectBank->setCurrentRow(idx);
 }
 
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
```

Comment:
I'd add a TODO comment here saying that much of this should be moved to the presenter. Note that all the changes in this PR are done in the view which means this functionality could not be tested at all in the automated tests if we wanted to do so.